### ISSUE TITLE
Bundler fail

### ISSUE THREAD

**Issue by rtrv** [mongoid/mongoid_search#113]
When I clone the source and just run `bundle install`, I get the following:

```
[!] There was an error parsing `Gemfile`: There are no gemspecs at /home/rtrv/mongoid_search. Bundler cannot continue.

 #  from /home/rtrv/mongoid_search/Gemfile:8
 #  -------------------------------------------
 #  platforms :jruby do
 >    gemspec :name => "mongoid_search-java"
 #  end
 #  -------------------------------------------
```

Gemspec name and gem name inside it are different. Is it my local issue or this JRuby dependenies gemspec requires some workaround?

UPD: Travis fails with another error:

```
[!] There was an error parsing `Gemfile`: There are multiple gemspecs at /home/travis/build/mongoid/mongoid_search. Please use the :name option to specify which one should be used. Bundler cannot continue.
 #  from /home/travis/build/mongoid/mongoid_search/Gemfile:4
 #  -------------------------------------------
 #  platforms :ruby do
 >    gemspec :name => "mongoid_search"
 #  end
 #  -------------------------------------------
```

**Comment by rtrv** [comment]
The only difference between gemspecs is stemmer gem:

```ruby
# For :ruby platform
s.add_dependency("fast-stemmer", ["~> 1.0.0"])

# For :jruby platform
s.add_dependency("jruby-stemmer", ["~> 0.0.1"])
```

`jruby-stemmer` is not in rubygems already! I suppose, it's ok to use `fast-stemmer` with jruby and keep just one .gemspec

**Comment by rtrv** [comment]
@sveredyuk, have you successfully run specs locally for Mongoid 6? I have working `bundle exec rake` for versions 3-5, but for 6 there is a problem with ActiveSupport.

**Comment by rtrv** [comment]
Actually fixed, the problem with JRuby is in #118
